VaticanBAV/et/BAVet119.xml

```diff
@@ -0,0 +1,569 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+   schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BAVet119" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+          <titleStmt>
+              <title xml:lang="gez" xml:id="t1">ዐውደ፡ ነገሥት፡</title>
```

Comment:
for manuscripts, we just provide a very short title (main text(s)) in English, the fidal/transcription/translation is what you need for works. And you do not even need the xml:id for the title in mss. It is the name of the manuscript, not of the text. 
It will not be processed by the app and the visualization will be wrong.